Sources/Kevin/Accounts/BankSelection/KevinBankSelectionView.swift

```diff
@@ -30,6 +30,12 @@ internal class KevinBankSelectionView : KevinView<KevinBankSelectionState> {
     
     private var state: KevinBankSelectionState?
     
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
```

Comment:
Maybe add a check for Dark mode check?

Reply:
Do we really need to check for dark mode here? We need to updated icons regardless if system transitions from Light to Dark or from Dark to Light